tooling/template-base-essentials/components/RenderLinks.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+const RenderLinks = (props: {
+  links: { [s: string]: unknown } | ArrayLike<unknown> | null;
+  linkStyling: string | undefined;
```

Comment:
Same as above for optional vs `string | undefined`. Will likely want to revisit these throughout.

Reply:
I have looked over all prop validations. Let me know if this can be improved further in any way.